core/ecschema-metadata/src/UnitConversion/Graph.ts

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+// Following https://github.com/dagrejs/graphlib/blob/master/lib/graph.js
```

Comment:
If we attribute something like this, I think we will need to have a copy of their license in our package.  There are examples within `core/frontend/ThirdPartyNotices.md`.

Reply:
I added a copy of their license to core/ecschema-metadata/ThirdPartyNotices.md